core/templates/core/base.html

```diff
@@ -7,6 +7,7 @@
         <meta id="utmCookieDomain" value="{{ analytics.UTM_COOKIE_DOMAIN }}" />
         {% include 'core/google_tag_manager_head.html' %}
         <meta charset="utf-8" />
+        <meta name="referrer" content="no-referrer">
```

Comment:
why are we doing this?

Reply:
django does not provide an out of the box way to add the "Referrer-Policy" header.

however, browsers support setting it via this `meta` approach.

So this is a valid but simpler way to achieve the same outcome.